<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registrar</class>
 <widget class="QMainWindow" name="Registrar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Recursos/indice-de-masa-corporal.png</normaloff>:/Recursos/indice-de-masa-corporal.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(111, 197, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(167, 217, 255);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QWidget" name="widget_3" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(158, 216, 255);</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Peso Min:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>kg</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>kg</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="inPesoMax"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>kg</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Peso Actual:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QPushButton" name="Calcular">
               <property name="text">
                <string>Calcular</string>
               </property>
               <property name="icon">
                <iconset resource="Recursos.qrc">
                 <normaloff>:/Recursos/calcular.png</normaloff>:/Recursos/calcular.png</iconset>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="inPesoMin"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Peso Max:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="inPeso"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="font">
                <font>
                 <family>Gadugi</family>
                </font>
               </property>
               <property name="text">
                <string>Altura:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="inAltura"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>m</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="0" colspan="3">
              <widget class="QPlainTextEdit" name="outHistorial">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="plainText">
                <string>                     - - Historial - -
-------------------------------------------------</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="outIMC">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>kg/m^2</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QLabel" name="label_13">
               <property name="font">
                <font>
                 <family>Gadugi</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>IMC(kg/m^2):</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Fecha:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLabel" name="outFecha">
               <property name="text">
                <string>/ / /</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(198, 218, 255);</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QWidget" name="widget_6" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="maximumSize">
                <size>
                 <width>125</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Recursos.qrc">:/Recursos/bmi.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="3">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string> |
 |
 |
 |
 |
 |</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="maximumSize">
                <size>
                 <width>6555</width>
                 <height>10562</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Microsoft YaHei</family>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Masa Corporal</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QWidget" name="widget_7" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="font">
                <font>
                 <family>Arial Rounded MT Bold</family>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>- - - - - - - - - - - - - - - - - - - - - - - - - - - - -</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionAbrir"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/agregar-archivo.png</normaloff>:/Recursos/agregar-archivo.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/salvar.png</normaloff>:/Recursos/salvar.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/abrir-correo-electronico.png</normaloff>:/Recursos/abrir-correo-electronico.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Recursos/boton.png</normaloff>:/Recursos/boton.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
